cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -151,6 +74,162 @@
+                                {{ form.topics }}
+                            </div>
+                        </div>
+                    {% endif %}
+                    {% if fields.value.authors %}
+                        <div class="form-l_col
+                                    form-l_col-1-2">
+                            <div class="form-group">
+                                <label class="form-label-header"
+                                       for="authors">
+                                    Author
+                                </label>
+                                {{ form.authors }}
+                            </div>
+                        </div>
+                    {% endif %}
+                    {% if fields.value.date_range %}
+                        <div class="form-l_col
+                                    form-l_col-1">
+                            <div class="form-group">
+                                <label class="form-label-header">
+                                    Date Range
+                                </label>
+                                <div class="form-l">
+                                    <div class="form-l_col
+                                                form-l_col-1-2">
+                                        <div class="form-group">
+                                            <label class="form-label-header"
+                                                   for="from_date">
+                                                From:
+                                            </label>
+                                            {{ form.from_date }}
+                                        </div>
+                                    </div>
+                                    <div class="form-l_col
+                                                form-l_col-1-2">
+                                        <div class="form-group">
+                                            <label class="form-label-header"
+                                                   for="to_date">
+                                                To:
+                                            </label>
+                                            {{ form.to_date }}
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    {% endif %}
+                </div>
+            </div>
+        </div>
+    {% endif %}
+{% endmacro %}
+
+{% macro _render_filter_fields(fields) -%}
+    {% if block.type == 'form' %}
+        <div class="form-l">
+            {{ loop(block.children) }}
+        </div>
+    {% elif block.type == 'email'
+       or block.type == 'number'
+       or block.type == 'password'
+       or block.type == 'tel'
+       or block.type == 'text'
+       or block.type == 'url' %}
+
+        <block type="{{ block.type }}"
```

Comment:
Sorry, I meant the elem, not the value of `type`. It's an `<input>` not a `<block>`